### DISCUSSION TITLE
Generating comments?

### DISCUSSION THREAD

**Discussion by pauldotknopf** [cdk8s-team/cdk8s#2003] [Q&A]
I'm using flux auto image updates, and it uses comments to declare where patches are applied.

Can I generate this comments with cdk8s?

<img width="908" alt="image" src="https://github.com/cdk8s-team/cdk8s/assets/523150/bde3ba66-6a9d-48dd-87ef-e7281917a8cb">

https://fluxcd.io/flux/guides/image-update/

**Comment by iliapolo** [comment]
Currently this is not possible. See https://github.com/cdk8s-team/cdk8s-core/issues/2573
